assert/assertions_test.go

```diff
@@ -148,8 +148,8 @@ func TestObjectsAreEqualValues(t *testing.T) {
 		{uint32(10), int32(10), true},
 		{0, nil, false},
 		{nil, 0, false},
-		{now, now.In(time.Local), true}, // should be time zone independent
-		{int(270), int8(14), false},     // should handle overflow/underflow
+		{now, now.In(time.Local), false}, // should not be time zone independent
```

Comment:
Did you mean to add the test here or in `TestObjectsAreEqual`. Although it fit in here also.